source/docs/software/advanced-controls/trajectories/troubleshooting.rst

```diff
@@ -121,7 +121,9 @@ If your feedforwards are bad then the P controllers for each side of the robot w
       :linenos:
       :lineno-start: 80
 
-2. (Java only) Next, we want to disable the Ramsete controller to make it easier to isolate our problematic behavior. This is a bit more involved, because we can't just set the gains (b and zeta) to 0. Pass the following into your ``RamseteCommand``:
+2. (Java only) Next, we want to disable the Ramsete controller to make it easier to isolate our problematic behavior. This is a bit more involved, because we can't just set the gains (b and zeta) to 0. Pass the following into your ``RamseteCommand``.
+
+.. note:: Teams using ``RamseteController`` directly can just pass ``0, 0``s into the constructor.
```

Comment:
This is also not true. In fact, the bullet point above this explicitly says not to do this.